components/authz-service/storage/postgres/postgres.go

```diff
@@ -37,12 +37,16 @@ func New(ctx context.Context, migConf migration.Config, dataMigConf datamigratio
 	return db, nil
 }
 
-func initPostgresDB(ctx context.Context, pgURL string) (*sql.DB, error) {
+func initPostgresDB(ctx context.Context, pgURL string, maxConnections int32) (*sql.DB, error) {
 	d, err := db.PGOpenContext(ctx, pgURL)
 	if err != nil {
 		return nil, err
 	}
 
+	if maxConnections > 0 {
+		d.SetMaxOpenConns(maxConnections)
+	}
```

Comment:
I'm wonder if we might get a bit more predictable performance by also bumping SetMaxIdleConns.

Reply:
what do you think about having them be the same

Reply:
For a small number of connections, I would advocate having them be the same.